satpy/readers/seviri_l2_bufr.py

```diff
@@ -19,13 +19,14 @@
 
 
 import logging
-from datetime import timedelta
+from datetime import timedelta, datetime
 import numpy as np
 import xarray as xr
 import dask.array as da
-
+from fnmatch import fnmatch
```

Comment:
module not needed, see comment on line 61

Reply:
removed